_examples/crons/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+)
+
+func runTask(monitorSlug string, duration time.Duration, shouldFail bool) {
+	checkinId := sentry.CaptureCheckIn(
+		&sentry.CheckIn{
+			MonitorSlug: monitorSlug,
+			Status:      sentry.CheckInStatusInProgress,
+		},
+		&sentry.MonitorConfig{
+			Schedule:      sentry.CrontabSchedule("* * * * *"),
+			MaxRuntime:    2,
+			CheckInMargin: 1,
+		},
+	)
+	task := fmt.Sprintf("Task[monitor_slug=%s,id=%s]", monitorSlug, *checkinId)
+	fmt.Printf("Task started: %s\n", task)
+
+	time.Sleep(duration)
+
+	var status sentry.CheckInStatus
+	if shouldFail {
+		status = sentry.CheckInStatusError
+	} else {
+		status = sentry.CheckInStatusOK
+	}
+
+	sentry.CaptureCheckIn(
+		&sentry.CheckIn{
+			ID:          *checkinId,
+			MonitorSlug: monitorSlug,
+			Status:      status,
+		},
+		nil,
```

Comment:
I don't know if this causes issues. In Laravel, we sent the monitor config along for all check-ins.

Reply:
It sorta feels natural to me: when the monitor already exists (guaranteed by the first check-in), we don't have to pass in the monitor config anymore.